### DISCUSSION TITLE
Can I add more website shortcuts to my Github profile?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23611] [New to GitHub]
<p>Hi, all ;D</p>
<p>For example, on my Github, on my user profile I only have 4 links.<br>
<img src="https://user-images.githubusercontent.com/10137/181078219-74ba4221-88b7-41d7-8820-df11e566b26e.png" alt="image" data-base62-sha1="hUz6gPxdQ0SoMTkw9qrVrbxSKX3" width="323" height="162"></p>
<h3>
<a name="question-1" class="anchor" href="#question-1"></a>question</h3>
<p>Is it possible to add more website shortcuts on Github?</p>
<p>For example, I would like to add the following links: LinkedIn/Repl.it/Codepen.io/Jsfiddle/JsBin/Glitch/Youtube/Medium</p>
<p>If anyone has any tips or has any knowledge that can help and clarify this question, I appreciate it.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p><a class="mention" href="/nkot56297">@nkot56297</a>, you can add more and more at your landing page, that is the special repo you want to create</p>
<p>create a new repo with the same username, in your case, it’s <code>nkot56297</code>, then you’ll see there the notification it’s a special repo,</p>
<p>just like how you edit any README, add your details</p>
<p>for example, mine,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/c/dcaa9d8a5d801a13ef661928dc669539c5f5278b.png" data-download-href="/uploads/short-url/vu6AhRkXMlfSO5Fgs2yX6OcqY3N.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078220-dd823eba-2f21-4a75-b224-bf7d3232c6a3.png" alt="image" data-base62-sha1="vu6AhRkXMlfSO5Fgs2yX6OcqY3N" width="690" height="438" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcaa9d8a5d801a13ef661928dc669539c5f5278b_2_690x438.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcaa9d8a5d801a13ef661928dc669539c5f5278b_2_1035x657.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcaa9d8a5d801a13ef661928dc669539c5f5278b_2_1380x876.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcaa9d8a5d801a13ef661928dc669539c5f5278b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1582×1006 88.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>now when you go back to your landing page, it will be displayed, you now have more of your details, just like this,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/3/0/30b960c93ce4ae2f3b0e2d652ef43dc99901b94c.png" data-download-href="/uploads/short-url/6X25x8D6pmy8ba04YySc4Q14FtO.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078221-d935268a-2cbb-436c-abeb-48f70c230f58.png" alt="image" data-base62-sha1="6X25x8D6pmy8ba04YySc4Q14FtO" width="690" height="407" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30b960c93ce4ae2f3b0e2d652ef43dc99901b94c_2_690x407.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30b960c93ce4ae2f3b0e2d652ef43dc99901b94c_2_1035x610.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30b960c93ce4ae2f3b0e2d652ef43dc99901b94c_2_1380x814.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/0/30b960c93ce4ae2f3b0e2d652ef43dc99901b94c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1711×1010 227 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>you can even see my example, check this out</p>
<p><a href="https://github.com/xdvrx1">https://github.com/xdvrx1</a></p>

**Comment by ghost** [comment]
<p>Thanks for all the information. I’m going to create this file in README and include all the links there.</p>
